includes/class-internal-events.php

```diff
@@ -113,17 +113,30 @@ public function is_internal_event( $action ) {
 	public function force_publish_missed_schedules() {
 		global $wpdb;
 
-		$missed_posts = $wpdb->get_col( $wpdb->prepare( "SELECT ID FROM {$wpdb->posts} WHERE post_status = 'future' AND post_date <= %s LIMIT 25;", current_time( 'mysql', false ) ) );
+		$page     = 1;
+		$quantity = 25;
+
+		do {
```

Comment:
Do we really need this loop here? The missed schedule event runs every minute so the next batch of 25 would just be published in the next ping? (Not to mention that something must be seriously wrong if we have more than 25 missed schedule posts.)